Update view-reports-for-mdo.md (#4555)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/view-reports-for-mdo.md

- **URL click protection action**: Shows the number of URL clicks by users in the organization and the results of the click:

  - **Blocked** (the user was blocked from navigating to the URL)

>   - **Blocked and clicked through** (the user choose to continue navigating to the URL)

Reviewer: ```suggestion
  - **Blocked and clicked through** (the user has chosen to continue navigating to the URL)
```

>   - **Clicked through during scan** (the user click on the link before the scan is complete)

Reviewer: ```suggestion
  - **Clicked through during scan** (the user has clicked on the link before the scan was complete)
```
